doc/user/content/sql/select.md

```diff
@@ -62,7 +65,7 @@ _join&lowbar;expr_ | A join expression; for more details, see our [`JOIN` docume
 **UNION** | Records present in `select_stmt` or `another_select_stmt`.<br/><br/>**DISTINCT** returns only unique rows from these results _(implied default)_.<br/><br/>With **ALL** specified, each record occurs a number of times equal to the sum of the times it occurs in each input statement.
 **INTERSECT** | Records present in both `select_stmt` and `another_select_stmt`.<br/><br/>**DISTINCT** returns only unique rows from these results _(implied default)_.<br/><br/>With **ALL** specified, each record occurs a number of times equal to the lesser of the times it occurs in each input statement.
 **EXCEPT** | Records present in `select_stmt` but not in `another_select_stmt`.<br/><br/>**DISTINCT** returns only unique rows from these results _(implied default)_.<br/><br/>With **ALL** specified, each record occurs a number of times equal to the times it occurs in `select_stmt` less the times it occurs in `another_select_stmt`, or not at all if the former is greater than latter.
-**AS OF** | If provided, `SELECT` will report the results at the supplied timestamp, meaning it reflects exactly those input updates at or before this timestamp.
+**AS OF** | If provided, `SELECT` will report the results at the supplied timestamp, meaning it reflects exactly those input updates at or before this timestamp. Note that `AS OF` [works differently](../tail#as-of) for `TAIL`, instead specifying the time at which a `TAIL` operation begins.
```

Comment:
Maybe it's just me, but I think of these as working the same. Both have their initial timestamp determined by `AS OF`, and it happens that `SELECT` just terminates. `SELECT` is basically `TAIL` which doesn't follow the diffs, and I think `AS OF` means the same things under that interpretation.

Reply:
My major concern is people finding the `TAIL` docs, since this was coming out of a customer who couldn't find them. I can just make it "See `TAIL` for discussion of `AS OF` in that context, or we can assume that the other mentions of `TAIL` I added are enough.

Reply:
I think linking to `TAIL` is great. My only problem is with announcing that it is different, as I think it is more the same than it is different. The only difference, imo, is that `TAIL` is different from `SELECT`; in both cases `AS OF` indicates when the query should be done, and `TAIL` continues from that point whereas `SELECT` does not.